package.json

```diff
@@ -39,15 +40,17 @@
     "gulp-eslint": "^2.0.0",
     "gulp-nodemon": "^2.0.3",
     "gulp-util": "^3.0.6",
+    "husky": "^0.14.3",
     "mongoose": "^4.4.8",
     "path": "^0.12.7"
   },
   "devDependencies": {
     "browser-sync": "^2.9.12",
-    "jsonlint": "^1.6.2",
     "gulp-tape": "0.0.7",
+    "jsonlint": "^1.6.2",
+    "supertest": "*",
     "tap-spec": "^4.1.1",
     "tape": "^4.2.2",
-    "supertest": "*"
+    "validate-commit-msg": "^2.14.0"
```

Comment:
This package is not well maintained, and I know we are using it on the main repo. But we should move away to something like commitlint, we havent done so yet on the main repo because the adaptor that we have for commitizen needs re-porting.

Lets hold on to this PR for now.